### PR TITLE
Add a sleep before re-checking clusteroperators

### PR DESCRIPTION
**Pull request by andfasano**
In some cases we observed a kube-apiserver restart after the first `oc wait clusteroperators --all --for=condition=Progressing=false --timeout=10m` completed successfully. 
While the investigation is continuing on that side, and since this scenario could cause some random test failures, this patch adds more waiting time before checking again the clusteroperators status.

@derekhiggins @bfournie

### CODE REVIEW

File: ci-operator/step-registry/baremetalds/e2e/test/baremetalds-e2e-test-commands.sh [hunk 1]
```diff
@@ -183,6 +179,13 @@ do
 done
 echo "$(date) - all imagestreams are imported."
 
+# additional check to ensure that all the co didn't switch back meanwhile
+echo "$(date) - Waiting 10 minutes before checking again clusteroperators"
+sleep 10m
```
Comment (derekhiggins) [inline ci-operator/step-registry/baremetalds/e2e/test/baremetalds-e2e-test-commands.sh:184]: The first wait may have taken 10 minutes (sometimes I think it works and does what we expect), maybe its better to add this sleep at the very start so we don't end up adding 20 minutes to the Job run time.

Comment (andfasano) [reply ci-operator/step-registry/baremetalds/e2e/test/baremetalds-e2e-test-commands.sh:184]: Yeah, probably we could just add the sleep at the very beginning and keep just one `oc wait`. I'd like to see now the output of some sample runs with this approach, we'll push the change later

Comment (andfasano) [reply ci-operator/step-registry/baremetalds/e2e/test/baremetalds-e2e-test-commands.sh:184]: I can see that there were some cases were 10 minutes timeout on the `oc wait` wasn't enough, so anticipating the sleep could also help for that

**Comment by derekhiggins** [comment]
@andfasano are the "-trigger" and "-dualstack" naming changes related?

**Comment by andfasano** [comment]
> @andfasano are the "-trigger" and "-dualstack" naming changes related?

No, it's just a "touch" on the test (but it could be anything else) to force rehearsal plugin to include it. One of the problems we had when testing changes against a step-registry step, it's that the potential set of jobs affected (across the several available versions) could be pretty large, and the rehearsal plugin chooses a limited subset.

**Comment by andfasano** [comment]
@derekhiggins as per the last discussions, even though we could spend 10 additional minutes per build, I'm feeling that the implementation with the two waits could be more robust, as we're going to check explicitly twice the operators status (instead of just one as per the current implementation). Wdyt?

**Comment by andfasano** [comment]
> > @derekhiggins as per the last discussions, even though we could spend 10 additional minutes per build, I'm feeling that the implementation with the two waits could be more robust, as we're going to check explicitly twice the operators status (instead of just one as per the current implementation). Wdyt?
> 
> Do you mean we should abandon this PR?
> 
> I think I'd prefer to put in a sleep for the moment (before the waits) as I've seen double finish in seconds, even in cases where the kubeapi hasn't finished restarting, e.g. https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/logs/periodic-ci-openshift-release-master-nightly-4.7-e2e-metal-ipi/1457544217147478016/artifacts/e2e-metal-ipi/baremetalds-e2e-test/build-log.txt

Between the waits (wait for co statuses to be ready, wait for some minutes, repeat the wait for cos to be ready)

**Comment by derekhiggins** [comment]
> > > @derekhiggins as per the last discussions, even though we could spend 10 additional minutes per build, I'm feeling that the implementation with the two waits could be more robust, as we're going to check explicitly twice the operators status (instead of just one as per the current implementation). Wdyt?
> > 
> > 
> > Do you mean we should abandon this PR?
> > I think I'd prefer to put in a sleep for the moment (before the waits) as I've seen double finish in seconds, even in cases where the kubeapi hasn't finished restarting, e.g. https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/logs/periodic-ci-openshift-release-master-nightly-4.7-e2e-metal-ipi/1457544217147478016/artifacts/e2e-metal-ipi/baremetalds-e2e-test/build-log.txt
> 
> Between the waits (wait for co statuses to be ready, wait for some minutes, repeat the wait for cos to be ready), please check commit [f536](https://github.com/openshift/release/pull/23419/commits/f536b54b73c55475c9626f6ff22e8d47aef5707e)

Ack, lgtm so, can you undo the CI job name changes once we're happy with the CI results and we can merge this.

**Comment by derekhiggins** [comment]
@andfasano looks read to merge, good to go?

**Comment by andfasano** [comment]
> @andfasano looks read to merge, good to go?

Agree, latest results seems good enough to proceed

**Comment by derekhiggins** [comment]
yup,
/lgtm

**Pull request merged**